developers/wcs/embeddings/index.md

```diff
@@ -1,293 +1,53 @@
 ---
 title: Weaviate Embeddings
-sidebar_position: 200
+sidebar_position: 0
 image: og/wcs/user_guides.jpg
 ---
 
-Weaviate Embeddings provides secure, scalable embedding generation as a fully managed service.
+Weaviate Embeddings provides secure, scalable embedding generation as a fully managed service. It integrates with Weaviate Cloud instances to generate, store, and search embeddings without managing infrastructure.
```

Comment:
I think this could be improved to clarify what it is (managed inference service), and what it does (generate embeddings). The second sentence almost sounds as though Weaviate Embeddings will also store & search, which I don't believe is the case.

I forget whether the mention of security was previously here. Is that something we want to highlight? Not sure whether there is anything in particular that makes it more secure than another inference service, for example.

Wdyt about:

```markdown
Weaviate Embeddings is a managed embedding inference service for Weaviate Cloud users. It helps generate embeddings for your data and queries conveniently and directly from a Weaviate Cloud database instance.
```

(This integrates the subsequent sentence)

Reply:
I agree; your wording is clearer and more to the point.